Scripts/script-GetDuplicatesMl.yml

```diff
@@ -79,7 +83,8 @@ script: |
           index = email.rfind("@")
           if index == -1:
               return None
-          return email[index + 1:]
+          return Utils.extract_domain_from_url(email[index + 1:])
```

Comment:
can't you have here out of bounds? in case @ is in the end?

Reply:
No, python slice are safe. it will return just empty string